### ISSUE TITLE
Bug: Translation file en.po changes ignored.

### ISSUE THREAD

**Issue by JamieRCHI** [MaFi-Games/Captain-of-Industry-issues#392]
**Describe the issue**
Changes to en.po are ignored.

**Expected behavior**
When various items are changed, while game not running, the nomenclature in the game should change the next time the game starts (or restarts). For example, I changed "Iron" to "Iron Plates"

code snippet from en.po: (see attached file)
```
#  unit product
msgid "Product_Iron__name"
msgstr "Iron Plates"   
#  was "Iron"
```

**To Reproduce**
1. Load attached save:   Gilligans Isle Modded 1.save
2. Load and replace en.po with attached file.
3. See attached Log file:   22-06-20_21-18-44_7789.log
4. Optionally install Mod at:   https://github.com/altmank/CaptainOfCheats (I don't think it is caused by this)
5. Load game and inspect storage next to dock that is filled with Iron (Should show Iron Plates.)

**Additional context**
 - Game version: v0.4.5
 - Note that only msgstr strings were changed in the en.po file.

**Attachments**
[Gilligans Isle Modded 1.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/8945952/Gilligans.Isle.Modded.1.zip)
[en.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/8945949/en.zip)
[22-06-20_21-18-44_7789.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/8945990/22-06-20_21-18-44_7789.zip)

**Comment by MaFi-Games** [comment]
We don't read en.po file for English souce strings, these are defined in the code.  And we don't plan to support it anytime soon.

**Comment by JamieRCHI** [comment]
There are some things in the English translation that I have issues with.
Some things are just plain wrong in my opinion. Part of the issue is that English in various countries is different. US, UK and other countries all different. 

If it is just a priority thing, I can live with that. As long as it is addressed sometime before final release I am also good with that. Thank you for your Consideration. 
(Maybe this would be labeled as a Low Priority, Postponed Thing? I hope it is not 'tabled' forever.)

And do I need to close this for now or leave it open?

**Comment by utdrmac** [comment]
I agree with JamieRCHI that there are indeed English grammatical errors, mainly in the descriptions of research and buildings. Has nothing to do with US/UK spelling but are plain wrong sentence structure. If we can't access the English translation files, how do you suggest we help?

**Comment by MaFi-Games** [comment]
We really appretiate your effor to improve our English source strings. But even if we would make en.po editable that wouldn't help as such changes would be visible only on your side. The best way would be to go to our translation tool https://translate.captain-of-industry.com/projects/captain-of-industry/game/  and place your siggestions into comments on the source english strings and once we go over that we can incorporate your feedback and take those improvements in.

**Comment by JamieRCHI** [comment]
I basically want to play around a bit on my game to get an idea on what works best on the User Interface for me. 
Like for instance, Using longer strings for the description of some things to see how they look in the game with the current user interface at various Screen Resolutions and Video Scale settings. 

Comments in the en.po file say that some strings need to be kept short. For example,  how many characters I can place in the text field for "Iron" at 1920x1080 resolutions and for 80, 90, or 100 percent scaling.

I believe this will make it easier for me to make more informed suggestions at the link you posted above.

**Comment by MaFi-Games** [comment]
One way to work-around this for local testing is to replace any other language with a same plural system (e.g. German) with en.po and then select German language and edit it.

**Comment by JamieRCHI** [comment]
Edit: Old file removed. Currently working on new one for Update 2.0.
I use this file for may English Grammer fixes. If anyone is interested.
For example:
U-shaped belts are called Sidewall belts in the US.
Cement Slabs should be called Cement Blocks (like the picture of the item depicts)
I use normal numbers instead of Roman numerals. Easier to read at a glance for many of us.
(You can ignore the 'Mind Control Towers' and 'Glass Rocking Chair' changes.)

Edit Again:  Would be nice if the parsing of the language files would ignore added comments in these files. This is causing multiple errors to show in the game Log files. This should never happen.

**Comment by JamieRCHI** [comment]
Verified that de.json can be edited and the changes show in the game. I'll post my latest 'english' de.json file. Note: Due to the large size of the file, I edit it with Notepad++. (I usually use JSONedit for this.)

[de.json](https://github.com/user-attachments/files/20496121/de.json)
